website_crm_quick_answer/data/automation.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
```

Comment:
IMO `<odoo noupdate="1">` and remove [this](https://github.com/OCA/website/pull/473/files#diff-f8cf199c897affda1478dd1864273340R3)

Reply:
done